### PR TITLE
Update to latest version, and a couple fixes/changes

### PR DESCRIPTION
**Pull request by christopherobin**
* Version has been updated to 0.7.2
* On some distros unzip is not available, added package directive to install it
* Added new variable to enable the consul UI in the systemd file

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 - name: "Create the consul group"
   group:
     name: consul
-    system: yes
```
Comment (MiLk) [inline tasks/main.yml]: You don't want consul to be a system group? :'(

File: templates/consul.systemd.j2 [hunk 2]
```diff
@@ -8,7 +8,8 @@ Environment="GOMAXPROCS=`nproc`"
 Restart=on-failure
 User=consul
 Group=consul
-ExecStart=/usr/local/bin/consul agent -config-dir /etc/consul
+ExecStart=/usr/local/bin/consul agent -config-dir /etc/consul{% if consul_ui_enabled|default(false)|bool %} -ui{% endif %}
```
Comment (MiLk) [inline templates/consul.systemd.j2]: I would prefer to have it in the config file https://www.consul.io/docs/agent/options.html#ui

**Review by MiLk — CHANGES_REQUESTED** [review]

File: tasks/main.yml (hunk 1 above)
Comment (christopherobin) [reply tasks/main.yml]: Oups sorry

File: templates/consul.systemd.j2 (hunk 2 above)
Comment (christopherobin) [reply templates/consul.systemd.j2]: Indeed, I'll move it

**Review by MiLk — APPROVED** [review]

**Pull request merged**